src/Builder/ListBuilder.php

```diff
@@ -169,14 +157,12 @@ public function buildActionFieldDescription(FieldDescriptionInterface $fieldDesc
     }
 
     /**
-     * @param string $type
-     *
-     * @return string
+     * @param int|string $type
```

Comment:
int accepted here?

Reply:
I used `int|string` because the call to this method is:
```php
$template = $this->getTemplate($fieldDescription->getType());
```
and `$fieldDescription->getType()` [returns `int|string`](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Admin/FieldDescriptionInterface.php#L116), I haven't dug if this is wrong and is not updated.